azure-devops/azext_devops/test/team/test_team.py

```diff
@@ -0,0 +1,126 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import unittest
+
+try:
+    # Attempt to load mock (works on Python 3.3 and above)
+    from unittest.mock import patch
+except ImportError:
+    # Attempt to load mock (works on Python version below 3.3)
+    from mock import patch
+
+from vsts.core.v4_0.core_client import CoreClient
+from azext_devops.dev.team.team import (create_team,
+                                        delete_team,
+                                        get_team,
+                                        get_teams,
+                                        get_team_members,
+                                        update_team)
```

Comment:
Given that all the functions from team.py are imported why not use * here??

from azext_devops.dev.team.team import *